tasks/deploy/config/base/aerodrome-lp-aero-usdc-91day.ts

```diff
@@ -0,0 +1,93 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { Address, keccak256, parseEther, toBytes, zeroAddress } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import {
+    AERO_USDC_GAUGE_ADDRESS_BASE,
+    AERO_USDC_LP_ADDRESS_BASE,
+    THREE_MONTHS,
+} from "../../lib/constants";
+import { BASE_AERODROME_LP_COORDINATOR_NAME } from "./aerodrome-lp-coordinator";
+import { BASE_FACTORY_NAME } from "./factory";
+
+// The name of the pool.
+export const BASE_AERODROME_LP_AERO_USDC_91DAY_NAME =
+    "ElementDAO 91 Day Aerodrome LP AERO-USDC Hyperdrive";
+
+const CONTRIBUTION = parseEther("100"); // 1e20
+
+export const BASE_AERODROME_LP_AERO_USDC_91DAY: HyperdriveInstanceConfig<"AerodromeLp"> =
+    {
+        name: BASE_AERODROME_LP_AERO_USDC_91DAY_NAME,
+        prefix: "AerodromeLp",
+        coordinatorAddress: async (hre) =>
+            hre.hyperdriveDeploy.deployments.byName(
+                BASE_AERODROME_LP_COORDINATOR_NAME,
+            ).address,
+        deploymentId: keccak256(
+            toBytes(BASE_AERODROME_LP_AERO_USDC_91DAY_NAME),
+        ),
+        salt: toBytes32("0x69420"),
+        extraData: AERO_USDC_GAUGE_ADDRESS_BASE,
+        contribution: CONTRIBUTION,
+        fixedAPR: parseEther("0.08"),
```

Comment:
Where'd you get this rate? Can you add a link that documents where you found this?

Reply:
I updated this to the emissions rate for the pool.  Here's the page:
https://aerodrome.finance/deposit?token0=0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913&token1=0x940181a94A35A4569E4529A3CDfB74e38FD98631&type=-1&position=0